src/openforms/forms/admin/views.py

```diff
@@ -126,6 +129,23 @@ class ExportSubmissionStatisticsView(
     # must be set by the ModelAdmin
     media: forms.Media | None = None
 
+    def form_valid(self, form: ExportStatisticsForm) -> HttpResponse:
+        start_date: date = form.cleaned_data["start_date"]
+        end_date: date = form.cleaned_data["end_date"]
+        dataset = form.export()
+        format = XLSX()
+        filename = f"submissions_{start_date.isoformat()}_{end_date.isoformat()}.xlsx"
```

Comment:
Maybe nice to add the form names, if the limit is used?

Reply:
I won't do that - this will not scale if you pick 10 forms for example.